frontend/package.json

```diff
@@ -79,12 +77,14 @@
     "@types/node": "12.12.17",
     "@types/pretty": "2.0.0",
     "@types/react": "16.9.16",
+    "@types/react-bootstrap-typeahead": "^3.4.6",
```

Comment:
Specify version

Reply:
Please only leave suggestions for the commits that I specified.